test/integration/scheduler_perf/config/performance-config.yaml

```diff
@@ -705,7 +705,7 @@
     mode: create
     templatePaths:
     - config/templates/pod-high-priority-large-cpu.yaml
-    intervalMilliseconds: 50
+    intervalMilliseconds: 200
```

Comment:
So, why did we have to increase the interval?

Reply:
I'm asking this because:
- The shorter `intervalMilliseconds` is, the more likely the test would make the improvement from the async preemption more visible.
- I saw the test case passed even after reverting this change to 50.

Reply:
The test running on test machines turned out to be much slower and the original rate of preemptions were starving out scheduling of regular pods completely.

You can use /test pull-kubernetes-scheduler-perf to test the performance on the test machines.

Reply:
I see, so `50` fails due to a weaker CI machine, even though my local machine is capable to handle.